djangoProject/user/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```

Comment:
nit: to make use of Django's built-in authentication functionality, it is recommended to extend the AbstractUser class